src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars__star {
+  display: flex;
+  place-content: center;
```

Comment:
The `display: flex;` property is unnecessary for `.stars__star` as it doesn't seem to be a flex container itself. The `place-content: center;` is not needed either since it's used to align flex items inside a flex container and `.stars__star` has no items inside.